articles/key-vault/vs-secure-secret-appsettings.md

```diff
@@ -97,18 +97,18 @@ If you are writing a quick prototype and don't want to provision Azure resources
 
 1. Install the following NuGet package to your project
     ```
-    Microsoft.Configuration.ConfigurationBuilders.Basic.1.0.0-alpha1.nupkg
+    Microsoft.Configuration.ConfigurationBuilders.UserSecrets.1.0.0-preview.nupkg
```

Comment:
-preview2 is the latest... although we hope to have preview3 out shortly, which will have some added features for UserSecrets in a VS scenario.